### PR TITLE
Properly clear mouse position indicator

### PR DESCRIPTION
**Pull request by flekschas**
## Description

What was changed in this pull request?

Properly clear mouse position indicator on mouseLeave, on blur, and when not hovering any track

Why is it necessary?

Before, the mouse position wasn't cleared which is annoying when there are multiple HiGlass instances as one always saw the mouse pos when leaving the root dom element which is uninformative.

## Checklist

- ~[ ] Unit tests added or updated~
- ~[ ] Documentation added or updated~
- ~[ ] Example added or updated~
- ~[ ] Screenshot for visual changes (e.g. new tracks)~
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: app/scripts/utils/show-mouse-position.js [hunk 1]
```diff
@@ -41,6 +41,8 @@ const showMousePosition = (
   // Graphics for cursor position
   const graphics = new PIXI.Graphics();
 
+  const clearMousePosition = () => { graphics.clear(); };
```
Comment (pkerpedjiev) [inline app/scripts/utils/show-mouse-position.js]: Why is this called `clearMousePosition`? It clears the graphics, not the mouse position.

File: app/scripts/utils/show-mouse-position.js [hunk 2]
```diff
@@ -50,7 +52,7 @@ const showMousePosition = (
    * @param  {Boolean}   isNoClear  If `true` do not clear the graphics.
    */
   const drawMousePosition = (mousePos, isHorizontal, isNoClear) => {
-    if (!isNoClear) graphics.clear();
+    if (!isNoClear) clearMousePosition();
```
Comment (pkerpedjiev) [inline app/scripts/utils/show-mouse-position.js]: To follow up on the comment above, why is it necessary to put the graphics.clear() call in a separate function. IMO it's a little misleading.

If you want to explain that the `graphics.clear()` call is used to clear the mouse position, I think a comment would be better suited.

Comment (flekschas) [reply app/scripts/utils/show-mouse-position.js]:
```suggestion
    if (!isNoClear) graphics.clear();
```

Leftover. Good catch.

File: app/scripts/utils/show-mouse-position.js (hunk 1 above)
Comment (flekschas) [reply app/scripts/utils/show-mouse-position.js]:
```suggestion
```

Leftover. Good catch.

**Comment by flekschas** [comment]
@pkerpedjiev Changed the name to `clearGraphics` :) Shall I merge it now if the TravisCI builds succeed or shall we wait first for v1.3 to come out?

**Comment by pkerpedjiev** [comment]
You can merge it now. It's a bug fix and not very risky.

**Pull request merged**